From b021b676c16c81d32a187a10c46206176882dc57 Mon Sep 17 00:00:00 2001
From: Thiraviyam Mariyappan <quic_tmariyap@quicinc.com>
Date: Tue, 28 Jan 2025 16:09:52 +0530
Subject: UPSTREAM:DP:wifi:mac80211: use 4 tx and rx queues

currently using one multi-queue for the Tx and Rx. Increase the queue
count to 4 since there are 4 TCL and REO rings for dedicated cores
and this improves approximatly one gbps improvement in TBTC DL.

Signed-off-by: Thiraviyam Mariyappan <quic_tmariyap@quicinc.com>
---
 net/mac80211/iface.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/mac80211/iface.c b/net/mac80211/iface.c
index 0a70db0..cab6e10 100644
--- a/net/mac80211/iface.c
+++ b/net/mac80211/iface.c
@@ -2381,7 +2381,7 @@ int ieee80211_if_add(struct ieee80211_local *local, const char *name,
 
 		ndev = alloc_netdev_mqs(size + txq_size,
 					name, name_assign_type,
-					ieee80211_if_setup, 1, 1);
+					ieee80211_if_setup, 4, 4);
 		if (!ndev)
 			return -ENOMEM;
 
-- 
2.34.1

